cli_tools/common/utils/storage/storage_client.go

```diff
@@ -31,7 +31,9 @@ import (
 )
 
 var (
-	gsRegex = regexp.MustCompile(`^gs://([a-z0-9][-_.a-z0-9]*)/(.+)$`)
+	bucket      = `([a-z0-9][-_.a-z0-9]*)`
+	bucketRegex = regexp.MustCompile(fmt.Sprintf(`^gs://%s/(.*)$`, bucket))
```

Comment:
Won't you need to make the '/' optional?

Reply:
ouch, I saw this comment after merging...
I didn't make '/' optional because upstream (gcloud) always pass in '/'. If I simply end it with "/?(.*)", then it may accept wrong bucket name. For example, that will pass "gs://bucket-name_mass".
We can make a more complex regex to be compatible with "gs://bucket-name" but so far I don't see the usage